cluster/manifests/skipper/skipper-redis.yaml

```diff
@@ -0,0 +1,45 @@
+{{ if eq .ConfigItems.skipper_clusterratelimit "true"}}
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  labels:
+    application: skipper-redis
```

Comment:
maybe call it `skipper-ingress-redis` just to be consistent with how we name it? :)

Reply:
done